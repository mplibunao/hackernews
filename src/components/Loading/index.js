import React from 'react';

const Loading = ()=>
  <i className="fa fa-spinner fa-spin"></i>

Loading.DefaultProps = {
  className: "fa fa-spinner fa-spin"
};

export default Loading ;