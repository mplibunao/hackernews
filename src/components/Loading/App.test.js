import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Loading from './';

describe('Loading', ()=>{
  
  it('renders', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Loading />, div);
  });

  test('snapshots', ()=>{
    const component = renderer.create(
      <Loading />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});