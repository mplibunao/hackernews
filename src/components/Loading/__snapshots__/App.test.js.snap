// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshots 1`] = `
<i
  className="fa fa-spinner fa-spin"
/>
`;
